docs/hacking.md

```diff
@@ -0,0 +1,264 @@
+<h1>Hacking</h1>
+
+# Development environment
+
+A `docker-compose.yml` file is provided to launch an OpenLDAP and a PostgreSQL
+instances as well as a phpLDAPAdmin to help you manage OpenLDAP.
+
+``` console
+$ docker-compose pull
+...
+Status: Image is up to date for dinkel/phpldapadmin:latest
+$ docker-compose up -d
+Creating network "ldap2pg_default" with the default driver
+Creating volume "ldap2pg_ldapetc" with default driver
+Creating volume "ldap2pg_ldapvar" with default driver
+Creating ldap2pg_ldap_1
+Creating ldap2pg_postgres_1
+Creating ldap2pg_admin_1
+```
+
+It's up to you to define how to access Postgres and LDAP containers from your
+host: either use DNS resolution or a `docker-compose.override.yml` to expose
+port on your host. Provided `docker-compose.yml`
+have [dnsdock](https://github.com/aacebedo/dnsdock) aliases
```

Comment:
Dans ce cas:
`docker-compose.yml` provides [dnsdock](https://github.com/aacebedo/dnsdock)  `postgres.ldap2pg.docker` and `ldap.ldap2pg.docker` aliases.